src/services/recentlyViewedProjects.js

```diff
@@ -47,17 +47,28 @@ angular.module("openshiftCommonServices")
         }
       });
 
-      // order by creationDate
-      projects =  $filter('orderObjectsByDate')(projects, true);
+      if (secondSort === 'displayName') {
```

Comment:
I'd rather just always use display name. Do we have a place where we need creation date? We should be consistent everywhere if possible.